api/conf/application.conf

```diff
@@ -1,4 +1,4 @@
 include "base.conf"
 
-db.default.url="jdbc:postgresql://localhost:5432/splashpagedb"
+db.default.url="jdbc:postgresql://vm:6049/splashpagedb"
```

Comment:
I think you want to leave this localhost as the VM will provide CONF_DB_DEFAULT_URL via configuration